### ISSUE TITLE
Jellyfin TV crashing upon startup, NullPointer

### ISSUE THREAD

**Issue by WibblyGhost** [jellyfin/jellyfin-androidtv#3209]
### Describe the bug

Getting crashes upon starting Jellyfin TV app, sometimes happens when trying to open the application and other times it happens after I select my user and login to my server. I have multiple users enabled and auto login disabled.

Around every second app launch triggers a crash so I can reliably trigger this. When I open the app it just immediately crashes, says an error has occured and sent logs to my server. However my server only shows a log saaying the WS socket was closed unexpectedly but no reasons.

~~Saw a similar issue here which looks related, but got closed down https://github.com/jellyfin/jellyfin-androidtv/issues/2947.~~

I've attached an Android ADB log and a Jellyfin server log.

### Logs

[Android ADB Crash Log](https://github.com/jellyfin/jellyfin-androidtv/files/13625514/JellyfinTVCrash_2.log)
[Android ADB Crash Log 2](https://github.com/jellyfin/jellyfin-androidtv/files/13625515/JellyfinTVCrash.log)

Jellyfin Log
```
[2023-12-10 07:59:00.816 +13:00] [INF] Defined LAN addresses : "[fc00::/7,fe80::/10,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]"
[2023-12-10 07:59:00.816 +13:00] [INF] Defined LAN exclusions : "[]"
[2023-12-10 07:59:00.816 +13:00] [INF] Using LAN addresses: "[fc00::/7,fe80::/10,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]"
[2023-12-10 07:59:45.546 +13:00] [INF] WS "::ffff:121.74.122.131" request
[2023-12-10 08:01:01.760 +13:00] [INF] Defined LAN addresses : "[fc00::/7,fe80::/10,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]"
[2023-12-10 08:01:01.760 +13:00] [INF] Defined LAN exclusions : "[]"
[2023-12-10 08:01:01.760 +13:00] [INF] Using LAN addresses: "[fc00::/7,fe80::/10,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]"
[2023-12-10 08:02:13.528 +13:00] [WRN] WS "121.74.122.131" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
[2023-12-10 08:02:13.544 +13:00] [INF] WS "::ffff:121.74.122.131" closed
[2023-12-10 08:02:48.272 +13:00] [INF] WS "::ffff:121.74.122.131" request
[2023-12-10 08:03:02.479 +13:00] [INF] Defined LAN addresses : "[fc00::/7,fe80::/10,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]"
[2023-12-10 08:03:02.479 +13:00] [INF] Defined LAN exclusions : "[]"
[2023-12-10 08:03:02.479 +13:00] [INF] Using LAN addresses: "[fc00::/7,fe80::/10,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]"
```

### Application version

0.15.12 Release

### Where did you install the app from?

Google Play

### Device information

TCL Smart TV Pro

### Android version

Android 11

### Jellyfin server version

10.8.13

**Comment by nielsvanvelzen** [comment]
Quickly glancing over those logs those all seem to be crashes that are fixed in 0.16. You could try joining the beta program or wait a few days for the stable release. Let me know if the crashes still occur after updating.

**Comment by WibblyGhost** [comment]
Sure I'll update to the beta release and see how it goes over the next couple of days. I'll close the issue if it runs with no issues.
